test/test_scanner.py

```diff
@@ -356,3 +356,13 @@ def test_systemd_239(self):
             # Check to make sure an older CVE isn't included
             "CVE-2017-9445",
             ])
+
+    def test_rsync_2_5_6(self):
+        """Scanning test-rsync-2.5.6.out"""
+        self._binary_test('test-rsync-2.5.6.out', 'rsync', '2.5.6', [
+            # Check for known cves in this version
+            "CVE-2007-6200",
+            "CVE-2007-6199",
+            ], [
+            # Check to make sure an older CVE isn't included
```

Comment:
Adding the Samba Rsync bugs may help you find a negative test here, but if there aren't any rsync bugs that don't affect this version, you could also use a CVE from another package as a negative test.